### PR TITLE
feat(component): add rows disabling for the Worksheet

### PR DESCRIPTION
**Pull request by bc-apostoliuk**
## What?
Add rows disabling for the Worksheet.

## Why?
To be able to disable a certain line of the worksheet

## Screenshots/Screen Recordings

Screenshot 2022-06-21 at 13 58 08

Disabling for expanded rows:
Screenshot 2022-06-21 at 17 58 19

Props:

Screenshot 2022-06-21 at 18 01 24

## Testing/Proof
Locally + UT

### CODE REVIEW

**Review by vdm-kr — APPROVED** [review]
LGTM

**Review by Meztimuro — APPROVED** [review]

**Review by chanceaclark — CHANGES_REQUESTED** [review]
Looks good, but could you also update [`WorksheetPropTable`](https://github.com/bigcommerce/big-design/blob/master/packages/docs/PropTables/WorksheetPropTable.tsx) with the new prop?

Also, could you just smoke test and drop a screenshot of how it works/looks with expandable rows?

File: packages/docs/pages/worksheet.tsx [hunk 1]
```diff
@@ -665,6 +665,66 @@ const WorksheetPage = () => {
...
+                        items={items}
+                        onChange={(items) => items}
+                        onErrors={(items) => items}
+                      />
+                    );
+                  }}
+                  {/* jsx-to-string:end */}
+                </CodePreview>
+              ),
+            },
```
Comment (MariaJose) [inline packages/docs/pages/worksheet.tsx:727]: Should we also add the new prop inside the `Props` panel to let the user know about it? Probably  under the `worksheet` tab?

Comment (bc-apostoliuk) [reply packages/docs/pages/worksheet.tsx:727]: @MariaJose Thanks for the comment)) yes, we should. I have already added the description for this prop, and updated the screenshots for the PR))

**Review by chanceaclark — APPROVED** [review]

**Comment by bc-apostoliuk** [comment]
@chanceaclark Thanks)) Already done, please take a look at the updated PR description.

**Review by MariaJose — APPROVED** [review]

**Pull request merged**